pkg/networkservice/chains/forwarder/server.go

```diff
@@ -122,7 +121,6 @@ func NewServer(ctx context.Context, name string, authzServer networkservice.Netw
 					// mechanisms
 					memif.NewClient(vppConn,
 						memif.WithChangeNetNS(),
-						memif.WithExternalVPP(),
```

Comment:
@glazychev-art  I like this simplification a lot... but could you help me understand what allowed us to drop the WithExternalVPP... I don't quite follow.

Reply:
@edwarnicke 
Honestly, I don't understand - why was it necessary at all. We use this option only for forwarder, and there is only one place where we refer to it: 
https://github.com/networkservicemesh/sdk-vpp/blob/main/pkg/networkservice/mechanisms/memif/common.go#L254-L256
As I understand, we try to figure out, is `currentNetNS` is equal to `targetNetNS`. 
But this condition is always false for forwarder - https://github.com/networkservicemesh/sdk-vpp/blob/main/pkg/networkservice/mechanisms/memif/common.go#L264 and `getNamespace` returns `u.Path, nil` anyway (with `ExternalVPP` option and without).

Locally everything works without this option. If there are any problems on `ci`, I think we can always revert these changes.